documentation/builders/event-devices.md

```diff
@@ -0,0 +1,134 @@
+# Event devices
+
+## Background
+Event devices are generic input devices that are exposed in ``/dev/input``.
+This includes USB peripherals (Keyboards, Controllers, Joysticks or Mouse) as well as potentially bluetooth devices.
+
+A specific usecase for this could be, if a Zero Delay Arcade USB Encoder is used to wire arcade buttons instead of using GPIO pins.
+
```

Comment:
I would not render the Markdown file as plain.

Reply:
The link has a `?plain=1` at the end. I don't think it's necessary, but it's not that important :)

Reply:
I thank he was referring to the addition of `?plain=1` at the end of the URL (your comment on 15-Aug)

Reply:
Ah sorry now I see it - sorry I was looking at the lines above and could not see anything.
Fixed now. Thanks for pointing this out.